index.html

```diff
@@ -0,0 +1,228 @@
+            <br />
+            project
+          </p>
+          <button class="hero__button">Download FREE!</button>
+        </div>
+        <img src="./images/screen.png" alt="picture" class="hero__photo" />
+      </div>
+    </header>
+
+    <main>
+      <section id="features" class="container features">
+        <header class="section features__header">
+          <h2>FEATURES</h2>
+        </header>
+
+        <div class="section features__cards">
+          <div class="card">
+            <img src="./images/responsive.svg" alt="" class="card__icon" />
+            <h3>FULLY RESPONSIVE</h3>
+            <p>
+              Looks amazing on any device:<br />
+              smartphone, tablet, laptop and<br />
+              deskop.
+            </p>
+          </div>
+
+          <div class="card">
+            <img src="./images/customizable.svg" alt="" class="card__icon" />
+            <h3>CUSTOMIZABLE</h3>
+            <p>
+              Change the colors, pictures or<br />
+              any of the sections to suit your<br />
+              needs.
+            </p>
+          </div>
+          <div class="card">
+            <img src="./images/design.svg" alt="" class="card__icon" />
+            <h3>SLICK AND BEAUTIFUL<br />DESIGN</h3>
+            <p>
+              Trendy and fresh design, fits<br />
+              any webiste.
+            </p>
+          </div>
+        </div>
+      </section>
+
+      <section class="container advertisement">
+        <img src="./images/logos.png" alt="advertisement-logos" />
+      </section>
+
+      <section class="container description-view">
+        <div class="section container-views">
+          <figure class="view">
+            <img src="./images/imac.png" alt="apple image" class="imac__image" />
+          </figure>
+          <article class="view article">
+            <h2>SIMPLE AND BEAUTIFUL</h2>
+            <p>
+              Use Nova theme for your next web project. It is<br />completely
+              customizable so you can change any of<br />
+              the sections to fit your needs. Nova Theme is Free for<br />any
+              kind of use, personal and commercial. Have fun<br />and good luck!
+            </p>
+          </article>
+        </div>
+      </section>
+
+      <section id="pricing" class="container pricing">
+        <header class="section pricing__header">
+          <h2>CHOOSE YOUR PRICING PLAN</h2>
+          <p>Pick any of our supper affordable pricing plans</p>
+        </header>
+        <div class="section pricing__cards">
+          <div class="pricing__card">
+            <h2 class="pricing__card--header">BASIC</h2>
+            <h2 class="pricing__card--price">$15</h2>
+            <p>per month</p>
+            <ul class="pricing__card--description">
+              <li>Up to 7 Projects</li>
+              <li>2 Additional Developers</li>
+              <br />
+              <br />
```

Comment:
Nigdy nie wykorzystujemy `br` do tworzenia odstępów - do tego jest `margin` oraz `padding` lub inna forma.
`br` to tylko łamanie linii tesktu ;)